HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -948,9 +948,11 @@ def to_oga(doc)
         XMLHelper.add_element(hvac_sizing_control, 'UseMaxLoadForHeatPumps', @use_max_load_for_heat_pumps, :boolean, @use_max_load_for_heat_pumps_isdefaulted) unless @use_max_load_for_heat_pumps.nil?
         XMLHelper.add_element(hvac_sizing_control, 'AllowIncreasedFixedCapacities', @allow_increased_fixed_capacities, :boolean, @allow_increased_fixed_capacities_isdefaulted) unless @allow_increased_fixed_capacities.nil?
       end
-      if not @schedules_filepath.nil?
+      if not @schedules_filepaths.nil?
```

Comment:
Related to the base-misc-defaults.xml comment, maybe needs to be `if (not @schedules_filepaths.nil?) && (not @schedules_filepaths.empty?)`